### PR TITLE
Apply to updated korean translation

### PR DESCRIPTION
**Pull request by crazyraven**
**What this PR does / why we need it**:
This is a my first time PR as a team-worker of Korean Translation team. This PR includes updated contents that translated to Korean. It's still not completed 100% yet, I hope that it could be approached our goal soon.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
I've uploaded new k8s.po, which is merged previous translated contents from @ianychoi.   

**Special notes for your reviewer**:
If you found any typo or mistranslation, suggesting appropriate word or substitution would be much appreciated.

**Release note**:

NONE

### CODE REVIEW

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -1,89 +1,2740 @@
...
+#: pkg/kubectl/cmd/create_rolebinding.go:35
+msgid ""
+"\n"
+"		  # Create a RoleBinding for user1, user2, and group1 using the admin "
+"ClusterRole\n"
+"		  kubectl create rolebinding admin --clusterrole=admin --user=user1 --user="
+"user2 --group=group1"
+msgstr ""
+"\n"
+"		 # admin ClusterRole 을 이용하여 user1, user2, 그리고 group1 의 ClusterRole 을 생성합니다\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: admin Role 을 이용하여 user1, user2, 그리고 group1 의 RoleBinding 을 생성합니다\n

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 2]
```diff
@@ -1,89 +1,2740 @@
...
+"10, no target CPU utilization specified so a default autoscaling policy will "
+"be used:\n"
+"		kubectl autoscale deployment foo --min=2 --max=10\n"
+"\n"
+"		# Auto scale a replication controller \"foo\", with the number of pods "
+"between 1 and 5, target CPU utilization at 80%:\n"
+"		kubectl autoscale rc foo --max=5 --cpu-percent=80"
+msgstr ""
+"\n"
+"		# deployment \"foo\" 에 대해 pod 의 개수 2 에서 10 사이로 오토스케일을 하도록 설정합니다. CPU 사용량이 "
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/사용량/사용률/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 3]
```diff
@@ -1,89 +1,2740 @@
...
+"latest version\n"
+"		# and print to stdout in json format.\n"
+"		kubectl convert -f pod.yaml --local -o json\n"
+"\n"
+"		# Convert all files under current directory to latest version and create "
+"them all.\n"
+"		kubectl convert -f . | kubectl create -f -"
+msgstr ""
+"\n"
+"		# 'pod.yaml' 을 최신 버전으로 전환하고 표준 출력으로 출력합니다.\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/표준 출력/표준출력/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 4]
```diff
@@ -1,89 +1,2740 @@
...
+"		# Apply the configuration in manifest.yaml and delete all the other "
+"configmaps that are not in the file.\n"
+"		kubectl apply --prune -f manifest.yaml --all --prune-whitelist=core/v1/"
+"ConfigMap"
+msgstr ""
+"\n"
+"		# pod.json 의 설정을 pod 에 적용합니다.\n"
+"		kubectl apply -f ./pod.json\n"
+"\n"
+"		# JSON 을 표준입력을 통하여 pod 에 적용합니다.\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/표준입력/표준 입력/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 5]
```diff
@@ -1,89 +1,2740 @@
...
+"		cat pod.json | kubectl create -f -\n"
+"\n"
+"		# docker-registry.yaml 의 데이터를 v1 API 형식을 이용하는 JSON 으로 편집하고 편집된 데이터를 이용하는 "
+"리소스를 생성합니다.\n"
+"		kubectl create -f docker-registry.yaml --edit --output-version=v1 -o json"
+
+#: pkg/kubectl/cmd/expose.go:53
+msgid ""
+"\n"
+"		# Create a service for a replicated nginx, which serves on port 80 and "
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: Better translate "replicated" too. it's not terminology :)

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 6]
```diff
@@ -1,89 +1,2740 @@
...
+"		# Describe pods by label name=myLabel\n"
+"		kubectl describe po -l name=myLabel\n"
+"\n"
+"		# Describe all pods managed by the 'frontend' replication controller (rc-"
+"created pods\n"
+"		# get the name of the rc as a prefix in the pod the name).\n"
+"		kubectl describe pods frontend"
+msgstr ""
+"\n"
+"		# node 의 세부정보를 표시합니다\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/node/노드/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 7]
```diff
@@ -1,89 +1,2740 @@
...
+"		# Create a service for a replicated nginx using replica set, which serves "
+"on port 80 and connects to the containers on port 8000.\n"
+"		kubectl expose rs nginx --port=80 --target-port=8000\n"
+"\n"
+"		# Create a service for an nginx deployment, which serves on port 80 and "
+"connects to the containers on port 8000.\n"
+"		kubectl expose deployment nginx --port=80 --target-port=8000"
+msgstr ""
+"\n"
+"		# 80 포트로 서비스 하고 container 의 8000 포트로 연결하는, replicated nginx 를 위한 service 를 "
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/service/서비스/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 8]
```diff
@@ -1,89 +1,2740 @@
...
+"\n"
+"		# 로컬에 임의의 포트를 열고 pod 의 5000 포트로 보냅니다\n"
+"		kubectl port-forward mypod 0:5000"
+
+#: pkg/kubectl/cmd/drain.go:118
+msgid "\n"
+"		# Mark node \"foo\" as schedulable.\n"
+"		$ kubectl uncordon foo"
+msgstr "\n"
+"		# \\\"foo\\\" 노드 상태를 schedulable 로 표시합니다.\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: \\\"foo\\\" => "foo"

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 9]
```diff
@@ -1,89 +1,2740 @@
...
+"		$ kubectl uncordon foo"
+
+#: pkg/kubectl/cmd/drain.go:93
+msgid "\n"
+"		# Mark node \"foo\" as unschedulable.\n"
+"		kubectl cordon foo"
+msgstr "\n"
+"		# \\\"foo\\\" 노드 상태를 unschedulable 로 표시합니다.\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: 위와 같음

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 10]
```diff
@@ -1,89 +1,2740 @@
...
+"		kubectl top pod --namespace=NAMESPACE\n"
+"\n"
+"		# Show metrics for a given pod and its containers\n"
+"		kubectl top pod POD_NAME --containers\n"
+"\n"
+"		# Show metrics for the pods defined by label name=myLabel\n"
+"		kubectl top pod -l name=myLabel"
+msgstr ""
+"\n"
+"		# 기본 namespace 의 모든 pod 에 대한 metrics 를 보여줍니다\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/기본/default/

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 11]
```diff
@@ -1,89 +1,2740 @@
...
+"		# Abort and reverse an existing rollout in progress (from frontend-v1 to "
+"frontend-v2).\n"
+"		kubectl rolling-update frontend-v1 frontend-v2 --rollback"
+msgstr ""
+"\n"
+"		# frontend-v1 의 pod 를 frontend-v2.json 의 새로운 replication conroller 데이터로 "
+"업데이트 합니다.\n"
+"		kubectl rolling-update frontend-v1 -f frontend-v2.json\n"
+"\n"
+"		# 표준 입력으로 들어오는 JSON 데이터를 이용하여 frontend-v1 의 pod 를 업데이트 합니다.\n"
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: `rolling-update` is not just simply `update`

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 12]
```diff
@@ -1,89 +1,2740 @@
...
+
+#: pkg/kubectl/cmd/taint.go:58
+#, c-format
+msgid ""
+"\n"
+"		Update the taints on one or more nodes.\n"
+"\n"
+"		* A taint consists of a key, value, and effect. As an argument here, it is "
+"expressed as key=value:effect.\n"
+"		* The key must begin
```
Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/stdin/표준 입력/

Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/Docker/도커/

Comment (yue9944882) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: s/주석/annotations/
thanks😃

**Review by yue9944882 — COMMENTED** [review]
Plz homogenise "service" with "서비스", "node" with "노드". Mixing use might be confusing.

**Review by ianychoi — COMMENTED** [review]
I addressed all except three things:

1) rolling-update: source string describes "update" after overall explanation on "rolling update", so I added translation on overall explanation
2) "디폴트" is better than "default"
3) Korean people use "Docker" term widely rather than "도커". 

For "annodations", Korean people do not specify plural "s" on explanatory strings. So I removed "s" by when reflecting @yue9944882 's comment.

Thanks a lot for comment and I would like to reflect as a co-author.

**Review by gochist — COMMENTED** [review]
@ianychoi I think "default" in the "default namespace" is not a subject to be translated. It is a name of namespace.

**Comment by yue9944882** [comment]
@ianychoi  @gochist  yeah, it's the name "default" not its meaning.

> "디폴트" is better than "default"
> Korean people use "Docker" term widely rather than "도커".

i'm okay with either of them. plz just don't switch between them :-)

**Comment by yue9944882** [comment]
@ianychoi thx. could u plz raise several smaller pulls next time? it's sort of hard to re 🤔

**Comment by yue9944882** [comment]
@crazyraven @ianychoi @gochist 수고하셨습니다👏🏻

**Comment by ianychoi** [comment]
@yue9944882 Definitely next time, thanks a lot for your so kind review!

**Comment by yue9944882** [comment]
/lgtm cancel

To mute bot rebuilding 😅

File: translations/kubectl/ko_KR/LC_MESSAGES/k8s.po [hunk 13]
```diff
@@ -1,89 +1,2828 @@
...
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.0.6\n"
-"X-Poedit-SourceCharset: UTF-8\n"
-"Language-Team: \n"
-"Plural-Forms: nplurals=1; plural=0;\n"
-"Language: ko_KR\n"
+"PO-Revision-Date: 2018-08-08 06:17+0000\n"
+"Last-Translator: Ian Y. Choi <ianyrchoi@gmail.com>\n"
+"Language-Team: Korean\n"
+"Language: ko\n"
```
Comment (gochist) [inline translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: I guess that this change from ```ko_KR``` to ```ko``` made the build failure.

Comment (ianychoi) [reply translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: Done - thanks!

Comment (gochist) [reply translations/kubectl/ko_KR/LC_MESSAGES/k8s.po]: It was just guessing. I don't know how to reproduce the automated verification in my local environment.

**Comment by yue9944882** [comment]
bumping this thread. ci doesn't look happy 😉ptal

**Pull request closed without merging**